common/src/main/java/com/revenuecat/purchases/common/networking/HTTPResult.kt

```diff
@@ -4,17 +4,24 @@ import org.json.JSONObject
 
 private const val SERIALIZATION_NAME_RESPONSE_CODE = "responseCode"
 private const val SERIALIZATION_NAME_PAYLOAD = "payload"
+private const val SERIALIZATION_NAME_ORIGIN = "origin"
+
+enum class ResultOrigin {
```

Comment:
`ResultOrigin` as a global name is a bit ambiguous. How about `HTTPResponseOrigin`?

Reply:
Agreed, I moved it inside `HTTPResult` and renamed to `Origin` so now the enum is `HTTPResult.Origin`. Lmk if you prefer the other way.

Reply:
Yeah that's perfect.